src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -51,76 +43,23 @@ public boolean hasNote() {
         return inputs.hasNote;
     }
 
-    public boolean atSetpoint() {
-        return inputs.currentAngle.isNear(
-                inputs.angleSetpoint, GripperConstants.TOLERANCE.in(Units.Percent));
-    }
-
-    public boolean isReversed() {
-        return Utils.normalize(inputs.currentAngle.in(Units.Radians))
-                > CommandGroupsConstants.WRIST_MIN_BACKWARDS_ANGLE.in(Units.Radians);
-    }
-
     public Command setRollerPower(double power) {
         return run(() -> io.setRollerMotorPower(power)).withName("set roller power");
     }
 
-    public boolean isGripperNearFoldedPosition() {
-        return new Translation2d(
-                                        GripperConstants.GRIPPER_LENGTH.in(Units.Meters),
-                                        new Rotation2d(inputs.currentAngle))
-                                .getY()
-                        + gripperHeight.in(Units.Meters)
-                < GripperConstants.GRIPPER_OUTTAKE_MIN_HEIGHT.in(Units.Meters);
-    }
-
-    public Command setRollerAndWrist(double rollerPower, MutableMeasure<Angle> wristAngle) {
-        return setRollerPower(rollerPower).raceWith(setWristPosition(wristAngle));
-    }
-
     public Command intake() {
-        return setRollerAndWrist(
-                        GripperConstants.INTAKE_POWER, GripperConstants.INTAKE_ANGLE.mutableCopy())
-                .withName("intake");
+        return setRollerPower(GripperConstants.INTAKE_POWER).withName("intake");
     }
 
     public Command outtake() {
-        return setRollerAndWrist(
-                        GripperConstants.OUTTAKE_POWER,
-                        GripperConstants.OUTTAKE_ANGLE.mutableCopy())
-                .withName("outtake");
-    }
-
-    public Command setWristPosition(MutableMeasure<Angle> angle) {
-        return Commands.runOnce(() -> io.setAngle(angle)).withName("set wrist position");
-    }
-
-    public Command scoreTrap() {
-        return Commands.sequence(
-                setWristPosition(GripperConstants.WRIST_TRAP_ANGLE),
-                setRollerPower(GripperConstants.TRAP_POWER));
-    }
-
-    public Command setWristPower(DoubleSupplier power) {
-        return run(() -> io.setAngleMotorPower(power.getAsDouble())).withName("set wrist power");
+        return setRollerPower(GripperConstants.OUTTAKE_POWER).withName("outtake");
```

Comment:
"intake" and "outtake" are ambiguous as they are relevant to the intake, too.

Reply:
It's not relevant to this branch, it was decided on before.

Reply:
Okay. But if only I could count all the times y'all made unrelated changes in your branches… 🙃